lib/puppet/util/windows/user.rb

```diff
@@ -185,105 +200,105 @@ class PROFILEINFO < FFI::Struct
-    :WinAccountDomainGuestsSid                    , 43,
-    :WinAccountComputersSid                       , 44,
-    :WinAccountControllersSid                     , 45,
-    :WinAccountCertAdminsSid                      , 46,
-    :WinAccountSchemaAdminsSid                    , 47,
-    :WinAccountEnterpriseAdminsSid                , 48,
-    :WinAccountPolicyAdminsSid                    , 49,
-    :WinAccountRasAndIasServersSid                , 50,
-    :WinNTLMAuthenticationSid                     , 51,
-    :WinDigestAuthenticationSid                   , 52,
-    :WinSChannelAuthenticationSid                 , 53,
-    :WinThisOrganizationSid                       , 54,
-    :WinOtherOrganizationSid                      , 55,
-    :WinBuiltinIncomingForestTrustBuildersSid     , 56,
-    :WinBuiltinPerfMonitoringUsersSid             , 57,
-    :WinBuiltinPerfLoggingUsersSid                , 58,
-    :WinBuiltinAuthorizationAccessSid             , 59,
-    :WinBuiltinTerminalServerLicenseServersSid    , 60,
-    :WinBuiltinDCOMUsersSid                       , 61,
-    :WinBuiltinIUsersSid                          , 62,
-    :WinIUserSid                                  , 63,
-    :WinBuiltinCryptoOperatorsSid                 , 64,
-    :WinUntrustedLabelSid                         , 65,
-    :WinLowLabelSid                               , 66,
-    :WinMediumLabelSid                            , 67,
-    :WinHighLabelSid                              , 68,
-    :WinSystemLabelSid                            , 69,
-    :WinWriteRestrictedCodeSid                    , 70,
-    :WinCreatorOwnerRightsSid                     , 71,
-    :WinCacheablePrincipalsGroupSid               , 72,
-    :WinNonCacheablePrincipalsGroupSid            , 73,
-    :WinEnterpriseReadonlyControllersSid          , 74,
-    :WinAccountReadonlyControllersSid             , 75,
-    :WinBuiltinEventLogReadersGroup               , 76,
-    :WinNewEnterpriseReadonlyControllersSid       , 77,
-    :WinBuiltinCertSvcDComAccessGroup             , 78,
-    :WinMediumPlusLabelSid                        , 79,
-    :WinLocalLogonSid                             , 80,
-    :WinConsoleLogonSid                           , 81,
-    :WinThisOrganizationCertificateSid            , 82,
-    :WinApplicationPackageAuthoritySid            , 83,
-    :WinBuiltinAnyPackageSid                      , 84,
-    :WinCapabilityInternetClientSid               , 85,
-    :WinCapabilityInternetClientServerSid         , 86,
-    :WinCapabilityPrivateNetworkClientServerSid   , 87,
-    :WinCapabilityPicturesLibrarySid              , 88,
-    :WinCapabilityVideosLibrarySid                , 89,
-    :WinCapabilityMusicLibrarySid                 , 90,
-    :WinCapabilityDocumentsLibrarySid             , 91,
-    :WinCapabilitySharedUserCertificatesSid       , 92,
-    :WinCapabilityEnterpriseAuthenticationSid     , 93,
-    :WinCapabilityRemovableStorageSid             , 94
+      :WinNullSid, 0,
```

Comment:
Please don't do this

Reply:
+1. It's much more readable before the change. We also use two-space code indent, and this would change it to four spaces, actually making it less consistent with the rest of the repo.

Formatting changes like these make it take longer to review PRs and make it harder to see the important functional changes that are happening. If ever there is an actual need for reformatting, it should happen in a separate commit or if there's a lot of changes involved, a separate PR entirely.

Reply:
Undid that block